woocommerce/woocommerce-gateway-stripe
# Use dummy data for billing address info if none is provided

closes #1770

If checkout fields editor or another plugin is used to hide billing address fields checkout will fail with UPE.
This is same fix as we used in [WCPay](https://github.com/Automattic/woocommerce-payments/pull/2853)

This does not prevent merchants from hiding fields that are used by billing address verification and thus breaking all checkout. But address and postal code verification are not enabled by default. Merchants will have to enable them in the dashboard and if they do I can hope they will not hide them from checkout. 

# Testing instructions

### Testing instructions 1
- Install checkout field editor.
- goto WooCommerce => Checkout fields.
- Hide billing_phone or/and billing_address_2 fields.
- Try checking out with UPE.
- Order should pass through, no errors observed.

### Testing instructions 2

- Install checkout field editor.
- goto WooCommerce => Checkout fields.
- Keep billing_phone or/and billing_address_2 fields but do not mark them as required.
- Try checking out with UPE.
- Order should pass through, no errors observed.

### Testing instructions 3

- Install checkout field editor.
- goto WooCommerce => Checkout fields.
- Keep billing_phone or/and billing_address_2 fields and mark them as required.
- Try checking out with UPE.
- Order should pass through, no errors observed.

-------------------
